code/__HELPERS/_lists.dm

```diff
@@ -1327,3 +1327,15 @@
 				&& deep_compare_list(log_1["stack"], log_2["stack"])
 		else
 			return TRUE
+
+
+/**
+ * Similar to pick_weight_recursive, except without the weight part, meaning it should hopefully not take
+ * up as much computing power for things that don't +need+ weights.
+ */
+/proc/pick_recursive(...)
```

Comment:
Also may want to include a usecase in the documentation for this because at first glance it can be used pretty strangely. 

Like you can invoke it as `pick_recursive(list(), list(), list())` or `pick_recursive(list(list(), list(), list())` and both work